File: VbConsolaJuego/VbConsolaJuego/PuntajePartidas.vb
Imports System.IO

Public Class PuntajePartidas


    Dim fileReader As StreamReader
    Public Sub New()

        Try

            fileReader =
            My.Computer.FileSystem.OpenTextFileReader("C:/Users/oscar/Desktop/Puntaje/puntaje.txt")
            Dim stringReader As String
            stringReader = fileReader.ReadLine()
            Console.WriteLine(stringReader)

        Catch ex As Exception

            Console.WriteLine("no se contro el archivo")

        End Try


    End Sub

    Sub Finalize()

        fileReader.Close()



    End Sub


End Class

File: CsConsolaJuegito/CsConsolaJuegito/PuntajePartidas.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.IO;

namespace CsConsolaJuegito
{
    class PuntajePartidas
    {

        StreamReader archivo = null;
        public PuntajePartidas()
        {
             
                try
                {
                    archivo = new StreamReader("C:/Users/oscar/Desktop/Puntaje/puntaje.txt");
                    string texto = archivo.ReadToEnd();

                    string[] valores = texto.Split(' ');
                    foreach (string x in valores)
                    {
                        Console.WriteLine(x);
                    }
                }
                catch (Exception e)
                {
                    Console.WriteLine("El archivo no fue encontrado");
                }
            
        }
        ~PuntajePartidas()
        {
            archivo.Close();
        }



    }
}
